src/app/(proper_react)/(redesign)/(public)/how-it-works/components/SectionOne.tsx

```diff
@@ -0,0 +1,155 @@
+            </TelemetryButton>
+          </div>
+        </div>
+      </div>
+      {/* STEP 1 */}
+      <div className={styles.step}>
+        <div className={styles.stepTextContainer}>
+          <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+            {l10n.getString("section-1-step-1-title")}
+          </h3>
+          <h3 className={styles.stepSubtitle}>
+            {l10n.getString("section-1-step-1-subtitle")}
+          </h3>
+          <div className={styles.privateInfoRow}>
+            <Image src={NameIcon} alt="name icon" />
+            <p className={styles.privateInfoText}>
+              {l10n.getFragment("section-1-step-1-name", {
+                elems: {
+                  name: <span className={styles.boldedText} />,
+                },
+              })}
+            </p>
+          </div>
+          <div className={styles.privateInfoRow}>
+            <Image src={LocationIcon} alt="location icon" />
+            <p className={styles.privateInfoText}>
+              {l10n.getFragment("section-1-step-1-location", {
+                elems: {
+                  location: <span className={styles.boldedText} />,
+                },
+              })}
+            </p>
+          </div>
+          <div className={styles.privateInfoRow}>
+            <Image src={DOBIcon} alt="dob icon" />
+            <p className={styles.privateInfoText}>
+              {l10n.getFragment("section-1-step-1-dob", {
+                elems: {
+                  dob: <span className={styles.boldedText} />,
+                },
+              })}
+            </p>
+          </div>
+          <p className={styles.stepBody}>
+            {l10n.getString("section-1-step-1-closing-text")}
+          </p>
+        </div>
+        <Image src={RemoveStep1} alt="step 1 screenshot" />
+      </div>
+      {/* STEP 2 */}
+      <div className={`${styles.step} ${styles.reversedStep}`}>
+        <Image src={RemoveStep2} alt="step 2 screenshot" />
+        <div className={styles.stepTextContainer}>
+          <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+            {l10n.getString("section-1-step-2-title")}
+          </h3>
+          <h3 className={styles.stepSubtitle}>
+            {l10n.getString("section-1-step-2-subtitle")}
+          </h3>
+          <p className={styles.stepBody}>
+            {l10n.getString("section-1-step-2-text-1")}
+          </p>
+          <p className={styles.stepBody}>
+            {l10n.getString("section-1-step-2-text-2")}
+          </p>
+        </div>
+      </div>
+      {/* STEP 3 */}
+      <div className={styles.step}>
+        <div className={styles.stepTextContainer}>
+          <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+            {l10n.getString("section-1-step-3-title")}
+          </h3>
+          <h3 className={styles.stepSubtitle}>
+            {l10n.getString("section-1-step-3-subtitle")}
+          </h3>
```

Comment:
Same Q as above, re: `<h3>+<h3>` vs `<h3>+<h4>`.

Reply:
h3's were in the Figma specs.